src/lib.rs

```diff
@@ -75,7 +77,7 @@ pub fn checkout(lock: Lock, force: bool) -> anyhow::Result<()> {
     utils::command(
         Command::new("git")
             .current_dir(destination_path.as_path())
-            .args(["checkout", "--force", lock.branch.as_str()]),
+            .args(["checkout", "--force", lock.branch.unwrap().as_str()]),
```

Comment:
Please use `.expect()` with a message (why panic?) instead of `.unwrap()`.

Reply:
Fixed, thanks!

Reply:
Though in this case it's better to use `anyhow::Error`.
We're using panics for catching developer bugs or going through an unreachable state (e.g. when `Option` is always `Some`), but for invalid user input we're using `Result`s and conventional error propagation.

(irrelevant anymore due to the field changed back to mandatory)

Reply:
Sure. Reverted back to mandatory arg.